dbms/programs/server/MySQLHandler.cpp

```diff
@@ -334,4 +366,26 @@ void MySQLHandlerSSL::finishHandshakeSSL(size_t packet_size, char * buf, size_t
 
 #endif
 
+const String show_table_status_replacement_query("SELECT "
```

Comment:
Extra space after SELECT

Reply:
Forward declaration is required in MySQLHandler.h

Reply:
done